swiftide/src/transformers/prompts/contextualize_code_chunk.prompt.md

```diff
@@ -0,0 +1,18 @@
+# Filtering Code Context
+Your task is to filter the given file context to the code chunk provided. The goal is to provide a context that is still contains the lines needed for understanding the code in the chunk whilst leaving out any irrelevant information.
```

Comment:
What do you expect it to strip?

Reply:
Definitions / imports that are not referenced from the chunk. I have actually tested this and tweaked it a bit to effectively do this with GPT 3.5.